packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -734,14 +745,16 @@ class C4DTableOfContents extends MediaQueryMixin(
                 return html`
                   <li
                     class="${itemClasses}"
+                    part="list-item list-item--active"
```

Comment:
Not all of these list items will be active. You need to condition `list-item--active` on wether or not the item is actually active.

Reply:
Done, as you suggested.